config/sacCer3.mapping.yml

```diff
@@ -1,374 +1,360 @@
 #Our own, special field, tells an interface which fields to run prefix queries on
 numericalFields:
```

Comment:
Same, this file is identical to the master branch in terms of indentation, only change is removal of the nearest field